lib/parsers.js

```diff
@@ -0,0 +1,141 @@
+const _ = require("lodash");
+
+function object(value) {
+  if (_.isObject(value)) { return value; }
+  if (_.isString(value)) {
+    try {
+      return JSON.parse(value);
+    } catch (e) {
+      throw new Error(`Couldn't parse provided value as object: ${value}`);
+    }
+  }
+  throw new Error(`${value} is not a valid object`);
+}
+
+function number(value) {
+  if (_.isNumber(value) && !_.isNaN(value)) {
+    return value;
+  }
+  if (_.isNumber(parseInt(value, 10)) && !_.isNaN(parseInt(value, 10))) {
```

Comment:
What about floats?

Reply:
Fair enough, didn't think about that. Will add support for floats